Makefile

```diff
@@ -83,53 +81,86 @@ install:
 	cp -Ra dbicdh "$(DESTDIR)"/usr/share/openqa/dbicdh
 
 
+.PHONY: check
+check: checkstyle test-with-database
+
 .PHONY: checkstyle
 checkstyle: test-shellcheck
 ifneq ($(CHECKSTYLE),0)
 	PERL5LIB=lib/perlcritic:$$PERL5LIB perlcritic lib
 endif
 
-.PHONY: test
-ifeq ($(TRAVIS),true)
-test: run-tests-within-container
-else
-test: checkstyle test-with-database
-endif
+.PHONY: phony_explicit
+phony_explicit: 
+
+.PHONY: test-with-db
+test-with-db: test-with-db-default
 
-.PHONY: test-unit-and-integration
-test-unit-and-integration:
-	OPENQA_CONFIG= prove ${PROVE_LIB_ARGS} ${PROVE_ARGS}
+test-with-db%: phony_explicit
+	@$(MAKE) $(subst test-with-db,test-with-database,$@)
 
 .PHONY: test-with-database
-test-with-database:
+test-with-database: test-with-database-default
```

Comment:
there is `test-with-database` and `test-with-db`? Why have two?

Reply:
To be able use the shorter one and do not break current environments which might use the longer one (e.g. spec file)

Reply:
As the spec file is within the same repo I suggest to just keep one version around. I don't see any need for changing the name so better keep the older name intact.

Reply:
No, I prefer two versions as I will need the shorter one. Feel free to remove the other later if you really need that.

Reply:
I don't see how you could **need** the shorter one.